### PR TITLE
Cookie banner

### PR DESCRIPTION
**Pull request by Davidjwkinghorn**

### CODE REVIEW

File: src/components/Shared/CookieBanner.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+
+class CookieBanner extends React.Component {
+
+	constructor(props) {
```
Comment (BradHN1) [inline src/components/Shared/CookieBanner.js:7]: You'll notice the tab indentation for most of the code is 2 spaces.  I think we should keep that fairly consistent. 
8 spaces is a lot.

Comment (Davidjwkinghorn) [reply src/components/Shared/CookieBanner.js:7]: I am using a single tab for indentation. I agree 8 spaces is a lot. Maybe it's something in my vs code settings.

**Comment by BradHN1** [comment]
I went through an issue involving tabs with Dmitriy.  I think it is important that your VSCode doesnt’ replace spaces with tabs, and the file ends up with the space characters not tabs.  If you look at most of the code files you can move cursor forward by one space at a time, but not on the CookieBanner.js file.

**Comment by Davidjwkinghorn** [comment]
I think I've fixed the issue. In VS Code I had to set "Tab Size" to 2 spaces (it used to be 4). I haven't encountered this issue before since detect indentation was on and I was only modifying existing files before this, I believe. When I created a new file it used my default settings of 4 spaces for tabs.

**Comment by Davidjwkinghorn** [comment]
I've also just added a cookie that remembers if a user has seen the cookie banner before so it only displays once.

**Review by BradHN1 — APPROVED** [review]
Looks good

**Pull request merged**